src/protocol.h

```diff
@@ -250,8 +250,8 @@ enum ServiceFlags : uint64_t {
     // Nothing
     NODE_NONE = 0,
     // NODE_NETWORK means that the node is capable of serving the block chain. It is currently
-    // set by all Bitcoin Core nodes, and is unset by SPV clients or other peers that just want
-    // network services but don't provide them.
+    // set by all Bitcoin Core non pruned nodes, and is unset by SPV clients or other peers
+    // that just want network services but don't provide them.
```

Comment:
If you're gonna update this maybe overhaul the text? The concept of "want network services but don't provide them" doesn't really make sense anymore (especially with NODE_NETWORK_LIMITED). Maybe just say "indicates peer is capable of serving the full historical block chain".